main.go

```diff
@@ -93,11 +96,37 @@ func showVendored(src *backvendor.GoSource) {
 }
 
 func main() {
-	flag.Parse()
-	if flag.NArg() < 1 {
-		fmt.Printf("Usage: %s path\n", os.Args[0])
+	progName := filepath.Base(os.Args[0])
+	log.SetFlags(0) // For typical stderr output of a program.
+
+	// Stop the default behaviour of printing errors and exiting.
+	// Instead, silence the printing and return them.
+	cli := flag.CommandLine
+	cli.Init("", flag.ContinueOnError)
+	cli.SetOutput(ioutil.Discard)
+	cli.Usage = func() {}
+
+	usage := func(flaw string) {
+		log.Printf("%s: %s\n", progName, flaw)
+		log.Fatalf("usage: %s [-help] [-importpath=toplevel] path\n", progName)
+	}
+	err := cli.Parse(os.Args[1:])
+	if err == flag.ErrHelp || *helpFlag { // Handle ‘-h’.
+		fmt.Printf("%s: help requested\n", progName)
```

Comment:
I think it would be helpful to see the usage here.

Reply:
Yes; done.